docker/development/Dockerfile

```diff
@@ -17,7 +17,8 @@ ARG BASE_IMAGE=ubuntu:18.04
 FROM ${BASE_IMAGE}
 
 ARG BUILD_TYPE=Release
-ARG PARFLOW_COMMIT=9c0b0f54acd6fbef2902c3d52a6bc82c285b335b
+#ARG PARFLOW_COMMIT=9c0b0f54acd6fbef2902c3d52a6bc82c285b335b
```

Comment:
Do we need to keep this? If so can we add a comment for context as to what this commit points to and why we are keeping it in VC?

Reply:
I removed the commented out PARFLOW_COMMIT tag and used the latest one with a comment about the parflow date of that commit.